docs/review_process.md

```diff
@@ -138,7 +138,7 @@ PR is selected for the merge only if:
 - PR is not a Draft.
 - PR has a green status (successful build).
 - PR doesn't have merge conflicts with `master` branch.
-- PR has 3 approved reviews (as described above).
+- PR has approved reviews (as described above).
```

Comment:
```suggestion
- PR has 2 approved reviews (as described above).
```

Or, "PR has an approved review (as described above)." ?

Reply:
I did not want a number so we dont miss it next time but I agree the phrasing it not perfect

My other idea was

```suggestion
- PR has been approved (as described above).
```

Reply:
@prince-chrismc 
I understand your PR's meaning.
Now I think your PR is adequate.